src/RoughTest.java

```diff
@@ -14,7 +15,7 @@ public static void main(String[] args) {
 
 
 
-        for( int i=0; i<5;i++){//
+        for( int i=0; i< tc.length;i++){//
             for(int j =0 ;j<tc[0].length;j++){
```

Comment:
This is probably ok, but you might want to use `j<tc[i].length`, just to ensure you are using the length of the current row you are looking at.